capstone/src/main/java/com/google/sps/servlets/BlobstoreUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public final class BlobstoreUtil {
+  /* Return BlobKey for image uploaded through form. */
+  public BlobKey getBlobKey(
```

Comment:
Can this method be static?

Reply:
I believe so, since it's not storing / persisting any data.